views/random-beers.hbs

```diff
@@ -0,0 +1,10 @@
+{{#each beer }}
+    {{!-- <div><img class="beer-pic" src="{{ this.image_url }}"></div>
+    <div> {{ this.name }} </div>
+    <div> {{ this.description }} </div>
+    <div> {{ this.tagline }} </div> --}}
+    {{ > beerpartial this }}
+    {{!-- <br>
+    <hr/>
+    <br> --}}
```

Comment:
random beer also needed some extra info, you can add the extra two here after the partial